packages/dds/tree/src/feature-libraries/modular-schema/schemaChangeFamily.ts

```diff
@@ -0,0 +1,278 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { assert } from "@fluidframework/common-utils";
+import { ChangeEncoder, ChangeFamily, JsonCompatibleReadOnly, ProgressiveEditBuilder } from "../../change-family";
+import { ChangeRebaser } from "../../rebase";
+import {
+    FieldSchema,
+    GlobalFieldKey,
+    LocalFieldKey,
+    SchemaData,
+    TreeSchema,
+    TreeSchemaIdentifier,
+} from "../../schema-stored";
+import { AnchorSet, Delta, FieldKey } from "../../tree";
+import { brand, Invariant, Multimap } from "../../util";
+import { FieldChangeHandler, FieldChangeMap, FieldChangeset, NodeChangeset } from "./fieldChangeHandler";
+import { FullSchemaPolicy } from "./fieldKind";
+
+export class SchemaChangeFamily implements
+    ChangeFamily<any, NodeChangeset>,
```

Comment:
We should not use `any` here. We should either have: a single concreate type here ex: `ModularEditor = Map<FieldKindIdentifier,unknown>` (or maybe something more specific than unknown, or maybe use an object with fields), or a generic type, ex: ModularEditor<TFieldKinds> or something like that.